diktat-rules/src/test/resources/test/paragraph4/null_checks/IfConditionBreakCheckTest.kt

```diff
@@ -74,12 +63,3 @@ fun foo() {
     }
 }
 
-fun foo() {
```

Comment:
Will your logic trigger when both `if` and `else` have `break`? And when `if` has more than two branches?

Reply:
Yes. Added tests.